maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -112,11 +119,17 @@
         if (parent == null) {
             throw new MojoFailureException("No <parent> found");
         }
-        if (!parent.getDependencyConflictId().equals("org.jenkins-ci.plugins:plugin:pom")) {
+        if (!PARENT_DEPENDENCIES.contains(parent.getDependencyConflictId())) {
             throw new MojoFailureException("Unexpected <parent> " + parent);
         }
-        if (new ComparableVersion(parent.getVersion()).compareTo(new ComparableVersion(MINIMUM_PARENT)) < 0) {
-            PomHelper.setProjectParentVersion(pom, MINIMUM_PARENT);
+        String minimum_parent;
+        if (parent.getDependencyConflictId().equals(JENKINS_POM)) {
```

Comment:
I was actually referring to introducing a variable for `parent.getDependencyConflictId()`. Never mind, it does not matter.